ark/analysis/spatial_analysis.py

```diff
@@ -187,7 +187,7 @@ def calculate_cluster_spatial_enrichment(all_data, dist_matrices_dict, included_
 
     for i in range(0, len(included_fovs)):
         # Subsetting expression matrix to only include patients with correct fov label
-        current_fov_idx = all_pheno_data.iloc[:, 0] == included_fovs[i]
+        current_fov_idx = all_pheno_data.loc[:, fov_col] == included_fovs[i]
         current_fov_pheno_data = all_pheno_data[current_fov_idx]
```

Comment:
should be able to do `for fov in included_fovs` (I don't see `i` being used elsewhere)

Reply:
Yeah I can change this.